content/docs/dvclive/outputs.md

```diff
@@ -0,0 +1,64 @@
+# Output folder structure
```

Comment:
Style: here and everywhere - we are using First Letter Capital style for the titles.